Refactored src/components/AddOn/* from jest to Vitest

AceHunterr opened the pull request:
**What kind of change does this PR introduce?**
Refactor

**Issue Number:**

Fixes #2788 

**Did you add tests for your changes?**
No

**Snapshots/Videos:**
This is the snapshot for the changed for `src/components/AddOn/core/AddOnStore/AddOnStore.test.tsx`:
![AddOnStorage](https://github.com/user-attachments/assets/25adb9f6-52ff-4d88-a883-79b2b17657fb)

The rest of the files that were asked to change in the issue were already updated with vitest when I did git pull... and I ran the tests on them and that came out to be successful... I deleted the redundant .test.* files.

**If relevant, did you update the documentation?**
No

**Summary**

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

palisadoes reviewed (changes requested):
Why were these files deleted
1. src/components/AddOn/core/AddOnEntry/AddOnEntry.test.tsx
2. src/components/AddOn/support/components/Action/Action.test.tsx

Why was this file edited? It is out of scope
1. src/components/CheckIn/tagTemplate.ts

You didn't fix all the files in the original issue

AceHunterr commented:
I deleted 

src/components/AddOn/core/AddOnEntry/AddOnEntry.test.tsx

src/components/AddOn/support/components/Action/Action.test.tsx

These files because in the issue thread I was told that I can delete them for good.


Secondly regarding rest of the files mentioned in the issue... those were already updated by someone else written in vitest.

palisadoes commented:
> I deleted
> 
> src/components/AddOn/core/AddOnEntry/AddOnEntry.test.tsx
> 
> src/components/AddOn/support/components/Action/Action.test.tsx
> 
> These files because in the issue thread I was told that I can delete them for good.
> 
> Secondly regarding rest of the files mentioned in the issue... those were already updated by someone else written in vitest.

Thanks,

Why weren't the deleted files replaced with `.spec.` files?

AceHunterr commented:
> > I deleted
> > src/components/AddOn/core/AddOnEntry/AddOnEntry.test.tsx
> > src/components/AddOn/support/components/Action/Action.test.tsx
> > These files because in the issue thread I was told that I can delete them for good.
> > Secondly regarding rest of the files mentioned in the issue... those were already updated by someone else written in vitest.
> 
> Thanks,
> 
> Why weren't the deleted files replaced with `.spec.` files?

They were they were... There were actually 5 file changes requested in the issue... out of which 4 were already converted from jest into vitest .... 

and for 2 `src/components/AddOn/core/AddOnRegister/AddOnRegister.test.tsx` and `src/components/AddOn/support/components/MainContent/MainContent.test.tsx` were already having the .spec.* only...

for the rest 2 there were both the .test.* and .spec.* file.... so I asked in the issue comment and therefore I deleted the .test.* file..

for the current scenario we have the .spec.* vitest files for all the 5 files :

![img1](https://github.com/user-attachments/assets/48a9862e-76dd-48f1-83d9-bdf43b764d58)
![img2](https://github.com/user-attachments/assets/d96a5716-026f-4056-a40a-afae5f61c9e9)
